python/foglamp/services/core/api/support.py

```diff
@@ -89,6 +91,42 @@
     return web.json_response({"bundle created": bundle_name})
 
 
+async def get_syslog_entries(request):
+    """ Returns a list of syslog trail entries sorted with most recent first and total count
+        (including the criteria search if applied)
+
+    :Example:
+        curl -X GET http://localhost:8081/foglamp/syslog
+        curl -X GET "http://localhost:8081/foglamp/syslog?limit=5"
+        curl -X GET "http://localhost:8081/foglamp/syslog?source=Storage"
+        curl -X GET "http://localhost:8081/foglamp/syslog?limit=5&source=Storage"
```

Comment:
can we please make source case insensitive

Reply:
done